.circleci/verbatim-sources/job-specs-custom.yml

```diff
@@ -451,3 +450,26 @@
               echo 'xcodebuild failed!'
               exit 1
             fi
+      - run:
+          name: Run Simulator Tests
+          no_output_timeout: "2h"
+          command: |
+            set -e
+            if [ ${IOS_PLATFORM} != "SIMULATOR" ]; then 
+              echo "not SIMULATOR build, skip it."
+              exit 0
+            fi
+            WORKSPACE=/Users/distiller/workspace
+            PROJ_ROOT=/Users/distiller/project
+            source ~/anaconda/bin/activate
+            #install the latest version of PyTorch and TorchVision
+            #TODO: How to install the nightly build?
```

Comment:
@ezyang is it possible to install a PyTorch nightly build on MacOS via `pip`? I searched a lil bit, didn't find the right command to do it. The goal here is to use a recent (up-to-date) version of PyTorch to generate a torchscript model for the iOS TestApp to test.

Reply:
Something like:

```
pip install --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

as suggested in https://pytorch.org/get-started/locally/